tests/unit/presenters/admin/indexation-list-item-presenter-test.php

```diff
@@ -3,27 +3,79 @@
 namespace Yoast\WP\SEO\Tests\Unit\Presenters\Admin;
 
 use Brain\Monkey;
+use Mockery;
+use Monkey\Functions\stubs;
+use Yoast\WP\SEO\Helpers\Indexable_Helper;
 use Yoast\WP\SEO\Presenters\Admin\Indexation_List_Item_Presenter;
 use Yoast\WP\SEO\Tests\Unit\TestCase;
 
 /**
  * Class Indexation_List_Item_Presenter_Test.
  *
  * @coversDefaultClass \Yoast\WP\SEO\Presenters\Admin\Indexation_List_Item_Presenter
- *
+ * @covers \Yoast\WP\SEO\Presenters\Admin\Indexation_List_Item_Presenter
  * @group presenters
  */
 class Indexation_List_Item_Presenter_Test extends TestCase {
 
+	/**
+	 * The indexable helper.
+	 *
+	 * @var Mockery\MockInterface|Indexable_Helper The indexable helper.
+	 */
+	protected $indexable_helper;
+
+	/**
+	 * The class under test.
+	 *
+	 * @var Indexation_List_Item_Presenter The Indexation_List_Item_Presenter being tested.
+	 */
+	protected $instance;
+
+	/**
+	 * Provides a working test class.
+	 *
+	 * @param int $count The total unindexed count.
+	 */
+	public function default_arrange( $count ) {
```

Comment:
Can you give this a more descriptive name. I have no clue what this mean. And you can make this `protected`
```suggestion
	protected function create_presenter( $count ) {
```

Reply:
this is based on the arrange / act / assert pattern of unit testing where many tests share a common setup - so, a default arrange method. In other words, it's a workaround to make it possible to run setup() _with_ parameters.